src/app/app/app-routing.module.ts

```diff
@@ -37,7 +37,7 @@ export const routes: Routes = [
 ];
 
 @NgModule({
-	imports: [RouterModule.forRoot(routes, { useHash: true })],
+	imports: [RouterModule.forRoot(routes, { useHash: true, relativeLinkResolution: 'legacy' })],
```

Comment:
Is this addition needed?

Reply:
It's not needed, it uses to access child routes better but there is no use for it in out project.
if you'd like to read about it it's here on the "Routes" section 
https://blog.ninja-squad.com/2020/11/11/what-is-new-angular-11.0/

Reply:
Yes I read about it. But if it's not needed, why is it in the code? Why not remove it from the code, and then it will give the default value, which (as you say) is ok for us.

Reply:
it's not longer in the code